scripts/galah/test.sh

```diff
@@ -1,17 +1,13 @@
 #!/bin/bash
 
-# First build the parent sample and save both raw and H5 to current directory
-if python build_parent_sample.py "./downloaded_galah" "." --num_processes 1 --tiny; then
-    echo "Build parent sample for GALAH successful"
-else
-    echo "Build parent sample for GALAH failed"
-    exit 1
-fi
+python3 download.py --tiny
```

Comment:
I think we have been using `python` throughout with the assumption that people would be working with an appropriate env.
```suggestion
python download.py --tiny
```